fuzz/fuzz_targets/cranelift-fuzzgen.rs

```diff
@@ -356,8 +382,17 @@ fn run_test_inputs(testcase: &TestCase, run: impl Fn(&[DataValue]) -> RunResult)
 }
 
 fuzz_target!(|testcase: TestCase| {
+    let mut testcase = testcase;
+    let fuel: Option<u8> = std::env::args()
+        .find_map(|arg| arg.strip_prefix("--fuel=").map(|s| s.to_owned()))
+        .map(|fuel| fuel.parse().expect("fuel should be a valid integer"));
+    let isa = testcase.build_isa(fuel).unwrap();
+    for i in 0..testcase.ctrl_planes.len() {
+        testcase.ctrl_planes[i].set_fuel(isa.flags().control_plane_fuel())
```

Comment:
... actually, now that I look at the CI failure, I think we can both eliminate the Cranelift setting (which is causing the failure because the "default settings" test is not updated) and just use `fuel` directly here. If we want a mode later where a setting can also set fuel, we can certainly add that, but this setting doesn't appear to be wired up to the control plane anywhere except here so I think it's best to just omit.

Reply:
You suggested to flesh out the text representation of the control plane right now, so I didn't fix the tests related to that yet. I have a commit pretty much ready for that purpose, although there will probably be a couple iterations on that as well until it fits nicely with the rest of the parsing.

I'm totally fine with keeping the PR open until that's done as well.

Reply:
To be honest I think at this point I'd rather merge this logically consistent piece (without the Cranelift setting at all, and without any text parsing) and then we can review further bits as they're done -- otherwise the patch gets a bit unwieldy.

Reply:
Ok, I removed everything except the part in cranelift-control and the reading of the CLI argument.